dbt_dry_run/test/node_runner/test_model_runner.py

```diff
@@ -78,6 +78,42 @@ def test_model_as_view_runs_create_view() -> None:
     assert node.compiled_sql in executed_sql
 
 
+def test_partitioned_incremental_model_declares_dbt_max_partition_variable() -> None:
+    DBT_MAX_PARTITION_DECLARATION = (
```

Comment:
For PEP8 this should be lower case `dbt_max_partition_declaration`

Then this is good to go 👍

Reply:
Only for global constants at the module or class level. Google's style guide has a pretty good list: https://google.github.io/styleguide/pyguide.html#316-naming

But this looks good now! Thanks for changing. Will merge and do a release when I have time this week